go.mod

```diff
@@ -101,6 +101,13 @@ require (
 	github.com/fatih/color v1.10.0
 	github.com/form3tech-oss/jwt-go v3.2.5+incompatible // indirect
 	github.com/ghodss/yaml v1.0.1-0.20190212211648-25d852aebe32
+	github.com/glinton/ping v0.1.4-0.20200311211934-5ac87da8cd96
+	github.com/go-logfmt/logfmt v0.4.0
+	github.com/go-ole/go-ole v1.2.1 // indirect
+	github.com/go-redis/redis v6.12.0+incompatible
+	github.com/go-sql-driver/mysql v1.5.0
+	github.com/go-stomp/stomp v2.0.6+incompatible
+	github.com/goburrow/modbus v0.1.0
```

Comment:
There seem to be quite some unrelated libraries. Did you run a `make tidy`? Do you really need those?

Reply:
done